packages/document/main-doc/docs/en/configure/app/source/disable-default-entries.mdx

```diff
@@ -23,9 +24,9 @@ export default defineConfig({
 });
 ```
 
-After turning off the default behavior, you need to use [`source.entries`](/configure/app/source/entries) to configure custom entries.
+After disabling the default behavior, you will need to use the [`source.entries`](/configure/app/source/entries) configuration to define custom entry points.
 
 :::warning
-We recommend using the directory convention provided by Modern.js to organize the code, so as to better use the functions of the framework and avoid some redundant configurations.
+We recommend organizing your code using the directory conventions provided by Modern.js to better utilize framework functionality and avoid some redundant configuration.
```

Comment:
```suggestion
We recommend organizing your code using the directory conventions provided by Modern.js to make better use of the framework's functionality and to avoid some redundant configurations.
```